test/performance/odp_ipsecfwd.c

```diff
@@ -622,28 +637,224 @@ static void drain_events(void)
 	}
 }
 
+static inline int process_ipsec_in(odp_packet_t pkts[], const odp_ipsec_sa_t sas[], int num,
+				   odp_packet_t pkts_out[])
+{
+	odp_ipsec_in_param_t param;
+	int left, sent = 0, num_out, ret;
+
+	memset(&param, 0, sizeof(param));
+	/* IPsec in/out need to be identified somehow, so use user_ptr for this. */
+	for (int i = 0; i < num; ++i)
+		odp_packet_user_ptr_set(pkts[i], NULL);
```

Comment:
This is not needed in the sync mode here nor in process_ipsec_out().

Reply:
Ditched in v3.